src/crab/bitset_domain.hpp

```
@@ -2,28 +2,26 @@
   private:
-    using bits_t = std::bitset<EBPF_TOTAL_STACK_SIZE>;
+    using bits_t = boost::dynamic_bitset<>;
     bits_t non_numerical_bytes;
 
   public:
-    BitsetDomain() noexcept { non_numerical_bytes.set(); }
-
-    // no performant move constructor to std::bitset, and therefore no copy-then-move for BitsetDomain
-    explicit BitsetDomain(const bits_t& non_numerical_bytes) noexcept : non_numerical_bytes{non_numerical_bytes} {}
-    BitsetDomain(const BitsetDomain& non_numerical_bytes) = default;
+    BitsetDomain() : non_numerical_bytes(thread_local_options.total_stack_size()) { non_numerical_bytes.set(); }
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: non_numerical_bytes

(cppcoreguidelines-pro-type-member-init)